Sources/SwiftProtobufPluginLibrary/Descriptor.swift

```diff
@@ -452,6 +576,10 @@ public final class Descriptor {
   /// order they are defined in the .proto file.
   public let reservedNames: [String]
 
+  /// True/False if this Message is just for a `map<>` entry.
+  @available(*, deprecated, message: "Please open a GitHub issue if you think functionality is missing.")
```

Comment:
Maybe some of these should get `renamed: options.mapEntry` annotations so they can just migrate to the new api?